scripts/check_sample_grads.py

```diff
@@ -158,7 +158,10 @@ def check_samples():
     mdp = MDP(spec['T'], spec['R'], spec['p0'], spec['gamma'])
     amdp = AbstractMDP(mdp, spec['phi'])
 
-    mem_params = get_memory('f', n_obs=amdp.n_obs, n_actions=amdp.n_actions, leakiness=0.2)
+    mem_params = get_memory('f',
```

Comment:
What is `'f'`? Can we make this more self-documenting?

Reply:
changed to "fuzzy"